### PR TITLE
Subscribers report

### PR DESCRIPTION
**Pull request by IshikaM19**
added:
- function that loads all subscriber emails 
- added to send_email to send an email to everyone in the subscribers list
- Added a try and except so that if the email is not verified then the email will not send 
- Added a ClientError

### CODE REVIEW

File: pipeline/schema.sql [hunk 1]
```diff
@@ -65,6 +65,19 @@ CREATE TABLE sale_event(
...
+
 INSERT INTO item_type(item_type) VALUES
     ('album'),
     ('track');
+
+
+INSERT INTO subscribers(subscriber_email) VALUES
+    ('trainee.ishika.madhav@sigmalabs.co.uk'),
+    ('trainee.cai.thomas@sigmalabs.co.uk'),
+    ('trainee.angelo.beleno@sigmalabs.co.uk');
```
Comment (caithomas6789) [inline pipeline/schema.sql:82]: Good to include all of our emails but might be best to remove that in the future so if someone else runs it, it won't connect our emails.

File: README.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ This project is a data-driven solution that regularly scrapes the latest sales f
 
 This projects implements an ETL pipeline that includes, scraping, data transformation and storage in a cloud-based environment. Python libraries such as BeautifulSoup and Pandas are used to extract and process data. Furthermore, AWS services, including Lambda and ECS facilitate the cloud-based infrastructure, ensuring scalability and reliability.
 
-## Ouputs
+## Outputs
```
Comment (IB64) [inline README.md:11]: nice spot!

File: report/report.py [hunk 3]
```diff
@@ -489,20 +504,18 @@ def send_email(report_file_path: str):
...
+    for subscriber in subscribers:
+        try:
+            client.send_raw_email(
+                Source='trainee.ishika.madhav@sigmalabs.co.uk',
+                Destinations=[subscriber],
+                RawMessage={
+                    'Data': message.as_string()
+                }
+            )
+        except ClientError:
```
Comment (caithomas6789) [inline report/report.py:517]: Nice error catching

File: report/static/style.css [hunk 4]
```diff
@@ -84,10 +84,10 @@ th{
   margin-right: auto;
 }
 h3.subtitle{
-  font-size: 24px;
+  font-size: 20px;
   text-align: center;
   color: black;
-  font-family: Arial;
+  font-family: serif;
```
Comment (caithomas6789) [inline report/static/style.css:90]: Some good format tweaking.

**Review by IB64 — APPROVED** [review]
nice 👍

**Review by caithomas6789 — APPROVED** [review]

**Pull request merged**